devolving_music/lib/song_scores.py

```diff
@@ -110,37 +123,72 @@ def get_quality_list(self,
 
         return final_quality_list
 
-    ###
-    # static methods can be moved into helper function
-    ###
     @staticmethod
     # sorts in ascending order
     def get_info_sort(
-            score_suite_list: List["ScoreSuite"]) -> List["ScoreSuite"]:
+            score_suite_list: List["ScoreSuite"],
+            get_informed=False,
+            informed_measure=1) -> List["ScoreSuite"]:
         """
-        Returns a list of song suites ordered by low information songs with the lowest information score 
-        are randomly shuffled before being added to the list
-        
+        Returns a list of song suites ordered by low information songs with the lowest information score
+        randomly shuffled before being added to the list
+
         """
         score_suite_list.sort(key=lambda sub: sub.info_score)
```

Comment:
```suggestion
        score_suite_list = sorted(score_suite_list, key=lambda sub: sub.info_score)
```

As is, it sorts the list in place. Function should avoid mutating their arguments.

Reply:
Yes but it's slightly faster to do this way and it saves memory.

Reply:
https://pythonsimplified.com/difference-between-sort-and-sorted-in-python/#Which_is_faster_sort_or_sorted

Reply:
![image](https://user-images.githubusercontent.com/22030302/186464556-caee0e84-e3fd-480f-b3f8-b57b8cfc9002.png)

Reply:
Sorting on the order of a few hundred elements is already basically instantaneous, a 5% speedup means nothing. Mutating a function's argument without that being an explicit part of that function's contract is a huge no-no and a good way to cause confusing bugs. 

In case it's not clear, this is basically what I mean:

```python
def get_smallest(numbers: list[int]):
  numbers.sort()
  return numbers[0]

my_number_list_whose_order_is_important = [8, 4, 10, 3, -1]

smallest = get_smallest(my_number_list_whose_order_is_important)

print(my_number_list_whose_order_is_important)  # [-1, 3, 4, 8, 10]
```

That's a really bad behavior that risks forcing a later developer to spend a lot of time debugging. Human time is worth more than CPU time, and hours of human time are much more important than microseconds of CPU time.

Reply:
Okay fine. But this is not a diff for this PR. I'll have to refactor and change .sort to sorted for all the sorting functions and verify that things don't break. I made a new issue and I'll do this on a new pull request. I don't think it is relevant for this one.